pkg/parser/mysql/locale_format.go

```diff
@@ -33,7 +149,8 @@ func formatENUS(number string, precision string) (string, error) {
 		buffer.Write([]byte{'0'})
 		position, err := strconv.ParseUint(precision, 10, 64)
 		if err == nil && position > 0 {
-			buffer.Write([]byte{'.'})
+			// Use style-defined decimal point
+			buffer.Write([]byte(style.DecimalPoint))
```

Comment:
```suggestion
			buffer.WriteString(style.DecimalPoint)
```

Maybe reduce unnecessary allocation